dockerfiles/examples/azure/values-storage-auto-azurefile.yaml

```diff
@@ -0,0 +1,16 @@
+# Overrides for azure storage in hpcc.
```

Comment:
I wonder if these should go in a separate subdirectory azure/config/... to group them together.

Reply:
perhaps, but then is might be less clear that subdirs are helm charts..
should the helm subchars go in a 'helm' subdir?

And I'm not sure about "values-storage-retained-azurefile.yaml" since the helm chart generates the correct Release named based values content in it's output...

Reply:
I propose putting helm charts in a "helm" subdir. and the value config examples under "config".
That will allow other examples to have multiple helm varieties if need be, and distinguishes between the subdirs that are helm charts vs the config subdir.

So for Azure we'd have:
```
examples/
-azure/
--helm/
---storage/
--config/
```

@ghalliday @richardkchapman thoughts?